controllers/apiAuth.js

```diff
@@ -0,0 +1,36 @@
+const bcrypt = require('bcrypt');
+const jwt = require('json-web-token');
+const User = require('../server/models').User;
+
+module.exports = {
+  login(req, res) {
+    User.findAll({ where: { username: req.body.username }, })
```

Comment:
You may be able to do `User.findOne` here to prevent the need for `user = user[0];` a couple lines later. (I think `findOne` returns the match if it exists, rather than an array)